src/ochubryk/hometask3/couples_of_elements.py

```diff
@@ -0,0 +1,36 @@
+# Дан список чисел. Посчитайте, сколько в нем пар элементов,
+# равных друг другу. Считается, что любые два элемента, равные
+# друг другу образуют одну пару, которую необходимо посчитать.
+# Входные данные - строка из чисел, разделенная пробелами.
+# Выходные данные - количество пар.
+# Важно: 1 1 1 - это 3 пары, 1 1 1 1 - это 6 пар.
+
+
+def count_couples(lst):
+    couples = 0
+
+    for number in set(lst):
+        couples += sum(range(lst.count(number)))
```

Comment:
O(n**2). 
Another idea: 
if we have n same numbers (we can count them using dict), we will have for first of them n-1 pairs, for second n-2 etc up to 1 for the pre-last. it's arithmetic progression :)